contracts/Otoken.sol

```diff
@@ -16,8 +16,8 @@ import {AddressBookInterface} from "./interfaces/AddressBookInterface.sol";
 contract Otoken is ERC20Initializable {
     using SafeMath for uint256;
 
-    /// @notice address of the AddressBook module
-    address public addressBook;
+    /// @notice address of the Controller module
+    address public controller;
```

Comment:
if we upgrade the controller, is there a way to set this again?

Reply:
Yeah, when we upgrade the controller, the controller address does not change, it is always the same proxy address, the only address that change is the implementation address which is all handled by the Proxy mechanism.

Reply:
can we test upgrading the controller with this?